### DISCUSSION TITLE
Controller

### DISCUSSION THREAD

**Discussion by AdamOndrejkovic** [symfony/symfony#43685] [Q&A] [answered]
Hi,

I am new to Symfony and working on a tutorial to understand how it works. At the moment I have a controller that should render a twig but I get a fatal error when I run it and I couldn't find a solution.

This is the error: 
Fatal error: Uncaught Error: Class "Symfony\Bundle\FrameworkBundle\Controller\AbstractController" not found in C:\Users\adamo\PhpstormProjects\SymfonyCRUD\my_project_name\src\Controller\ArticleController.php:9 Stack trace: #0 {main} thrown in C:\Users\adamo\PhpstormProjects\SymfonyCRUD\my_project_name\src\Controller\ArticleController.php on line 9

Here is my controller
```
<?php
    namespace App\Controller;

    use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
    use Symfony\Component\HttpFoundation\Response;
    use Symfony\Component\Routing\Annotation\Route;
    use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;

    class ArticleController extends AbstractController{
    /*
     * @Route("/")
     * @Method({"GET"})
     */
    public function index():Response{
        return $this->render('articles/index.html.twig');
    }
}
```

Thanks for the help in advance.

**Comment by ahundiak** [comment, marked as answer]
The error message indicates it cannot find the AbstractController class.  You might try a `composer dump-autoload' to see if it is a autoloading issue.  You can also look under vendor/symfony/framework-bundle/Controller to verify the class file does indeed exist.

How did you create your Symfony app?

Be aware that many tutorials are badly outdated or just plain wrong.  Following the getting started section in the docs is your best bet.

**Reply by AdamOndrejkovic** [reply]
I've tried the command but it still doesn't work. Regarding the app, I've created it through the command line and when I look at the file it's there but for some reason, it shows an error.

**Reply by AdamOndrejkovic** [reply]
I've made it work I followed the steps from documentation and I've used the build-in view which a guess caused the problem because after I've run Symfony server:start.
